### PR TITLE
implement a flag for paying out Gratipay 1.0 balances

### PR DESCRIPTION
**Pull request by whit537**
For use on #3481.

### CODE REVIEW

**Comment by whit537** [comment]
Ready for review, @rohitpaulk, et al. We need this for tomorrow's payday.

File: gratipay/billing/payday.py [hunk 1]
```diff
@@ -408,23 +408,22 @@ def payout(self):
...
             SELECT p.*::participants
-              FROM participants p
-             WHERE balance > 0
-               AND ( SELECT count(*)
-                       FROM exchange_routes r
-                      WHERE r.participant = p.id
-                        AND network = 'balanced-ba'
-                   ) > 0
+              FROM exchange_routes r
+              JOIN participants p ON p.id = r.participant
```
Comment (rohitpaulk) [inline gratipay/billing/payday.py]: If a user had 4 old bank accounts and 1 active one, wouldn't this result in multiple calls to `ach_credit` with the same participant? Maybe changing this to `current_exchange_routes` would solve the problem?

**Comment by whit537** [comment]
Blech, forgot the UI for this. :)

File: gratipay/billing/payday.py (hunk 1 above)
Comment (whit537) [inline gratipay/billing/payday.py]: Does the same bug afflict MassPay?

Comment (rohitpaulk) [inline gratipay/billing/payday.py]: If it does - that's a serious bug, we could end up paying people double their balance :/ Here (ach payouts in payday), the worst case scenario is a few extra API calls.

Comment (rohitpaulk) [inline gratipay/billing/payday.py]: > we could end up paying people double their balance

```SQL
SELECT COUNT(*)
  FROM participants p
 WHERE (
 SELECT COUNT(*)
   FROM exchange_routes r
  WHERE network='paypal'
    AND r.participant = p.id
) > 1; -- returns 0
```
Hasn't happened till now.

Comment (rohitpaulk) [inline gratipay/billing/payday.py]: I think replacing `exchange_routes` with `current_exchange_routes` in both places should solve the problem.

**Comment by whit537** [comment]
Okay @rohitpaulk, how does it look now?

**Comment by rohitpaulk** [comment]
Looks good now.

**Comment by whit537** [comment]
Okay! I'll let you merge when Travis is green. Diving in on #3538 ...

**Comment by whit537** [comment]
Gah, still forgot the UI for this. :)

**Comment by whit537** [comment]
New PR for that: #3542.

**Pull request merged**